### ISSUE TITLE
Issue viewing forte.buzl.uk outside of Firefox

### ISSUE THREAD

**Issue by JaxzKyd** [kaangiray26/forte#92]
So far for my testing I've been using Firefox Nightly on my PC and phone to access forte.buzl.uk, and I went to try it on Microsoft Edge on my PC really quickly and after logging in it would show any songs or load the profile. Thought it strange then hopped on my phone to try Chrome, then Samsung Internet, then Brave, each one failed but the logs on the server don't indicate anything out of the ordinary
Snippet from the logs:
```
FhnLY2ViZ1PJPQd7e1EoFDanjb_325uf Session request.
lQpnTI2fA3gKaphLAiEkn2rImqaxSOgy /api/random/tracks
T5ashhkNwG9eXrSlEVwW9UeH9uw-duGB /api/profile
ql3q814_HPCtueEm6cKsv1i8YFtlJsQe /api/random/tracks
EQY5351bt6_FCAIOJxfqdTInYe7KYoE9 Session request.
mFtSXXUVquK_AS-epfJ-MeSOmeONHwsK /api/random/tracks
EOFpn-PAiqIe5CDJoQJ6D_Ht9GZsNOTZ Session request.
EfvWW3ugGJUCt7MI_rciLzkVYyr6mBFr /api/random/tracks
Sc7J5x1x90J3sXyJuvze9NDTyHrUyLpW Session request.
OHaMCBdSm7HFGeJ8niwQyfOIO6FFDzEt Session request.
pLY0ZOFwOHCia2-K3uCKUs2KcDKiFm5r Session request.
uA7MzDI5EBnm-1sJoKFkFVSf1KuRhU6Y /api/random/tracks
WFbgSl44mOkSyGBv6QKbbzvMV38DA-gw Session request.
nC5bD2f0evqVucEbdzwmnuxSt3tnhVPx /api/random/tracks
NvpXPqO3Gil4cEOcZphcUWWUZxGGcrqT /api/profile
```

**Comment by kaangiray26** [comment]
Hi, do you have any updates on this issue? Have you tried the latest version, forte-2.5?

What do you get in the developer console on your browser?

**Comment by JaxzKyd** [comment]
Have just tried the latest version, still no give. Here are some developer console logs from Edge
![image](https://user-images.githubusercontent.com/39567321/222576282-52187232-a21b-492c-8c08-ae645afc56e0.png)

**Comment by kaangiray26** [comment]
Can you maybe create an account for me, so that I can try it on your server? You can text me in private through email if you want.

**Comment by JaxzKyd** [comment]
Email sent 👍

**Comment by kaangiray26** [comment]
Hey, please make sure you've reset your caches, localStorage and sessionStorage on the devices you're trying to connect. I've pushed a new release which makes things easier to be used with nginx as a reverse proxy. I've checked the setup and I can confirm that forte is working behind a reverse proxy with a SSL certificate and with my android device using forte on Chrome browser.

**Comment by JaxzKyd** [comment]
I cleared cache, I tried to login on the local server machine (so I could do localhost:3000 instead of my domain to eliminate any reverse proxy issues) Still running into the same problem 😦 
![image](https://user-images.githubusercontent.com/39567321/222860901-c72b8fa4-7a1b-4a7d-bb91-dc8606a0d432.png)

**Comment by kaangiray26** [comment]
Hi, this issue seems to be fixed with the latest release of forte (forte-2.8) and the forte client. I've checked and can confirm it's working on chrome. The problem was caused by `credentials: "include"` by fetch API. Chrome and other browsers behave differently while making calls to a http resource from a https context. I've changed the API to apply the session via URL argument.

Hope this works for you too!
